src/ui/components/Passport/Passport.tsx

```diff
@@ -116,54 +116,72 @@ const Passport = () => {
       imageBaseName: "search_source_text",
     },
     {
-      label: "Jump to event",
-      completed: !showJumpToEvent,
-      videoUrl: "https://vercel.replay.io/passport/jump_to_an_event.gif",
-      imageBaseName: "jump_to_event",
-    },
-    {
-      label: "Jump to code",
-      completed: !showJumpToCode,
-      videoUrl: "https://vercel.replay.io/passport/jump_to_code.gif",
-      imageBaseName: "jump_to_code",
+      label: "Use focus mode",
+      completed: !showUseFocusMode,
+      videoUrl: "https://vercel.replay.io/passport/use_focus_mode.gif",
+      imageBaseName: "use_focus_mode",
     },
+  ];
+
+  const multiplayerItems = [
     {
       label: "Add a comment",
       completed: !showAddComment,
       videoUrl: "https://vercel.replay.io/passport/add_a_comment.gif",
       imageBaseName: "add_a_comment",
     },
     {
-      label: "Add a unicorn badge",
-      completed: !showAddUnicornBadge,
-      videoUrl: "https://vercel.replay.io/passport/unicorn_badge.gif",
-      imageBaseName: "add_a_unicorn_badge",
+      label: "Share",
+      completed: !showShareNag,
+      videoUrl: "https://vercel.replay.io/passport/share.gif",
+      imageBaseName: "share",
     },
+  ];
+
+  const sections = [
     {
-      label: "Use focus mode",
-      completed: !showUseFocusMode,
-      videoUrl: "https://vercel.replay.io/passport/use_focus_mode.gif",
-      imageBaseName: "use_focus_mode",
+      title: "Basics",
+      items: timeTravelItems,
     },
     {
-      label: "Record a replay",
-      completed: !showRecordReplay,
-      videoUrl: "https://vercel.replay.io/passport/record_a_replay.gif",
-      imageBaseName: "record_a_replay",
+      title: "Advanced",
+      items: powerToolsItems,
     },
     {
-      label: "Inspect element",
-      completed: !showInspectElement,
-      videoUrl: "https://vercel.replay.io/passport/inspect_an_element.gif",
-      imageBaseName: "inspect_element",
+      title: "Collaboration",
+      items: multiplayerItems,
     },
   ];
 
-  const selectedItem = updatedChecklistItems[stepIndex];
+  const selectedItem = sections[selectedIndices.sectionIndex].items[selectedIndices.itemIndex];
+
   const rand = (min: number, max: number) => {
     return Math.floor(Math.random() * (max - min + 1) + min);
   };
 
+  const renderSection = (section: any, sectionIndex: number) => {
```

Comment:
:hand: Avoid use of `any` unless _absolutely_ necessary.  Best approach here is to declare a TS type for the section objects, such as `interface Section { title: string, items: ItemTypeHere[]}`, and declare `const sections : Section[] = [/* section items here */]`.  Then you can declare this argument as `section: Section`.